lms/static/js/utils/dropdown.js

```diff
@@ -0,0 +1,235 @@
+var edx = edx || {},
+
+    Dropdown = (function() {
+
+        var _dropdown = {
+
+            opts: {
+                page: $(document),
+                midpoint: ($(window).width() / 2),
+                button: $('.button-more.has-dropdown'),
+                button_active: 'is-active',
+                menu: $('.dropdown-menu'),
+                menu_inactive: 'is-hidden',
+                menu_active: 'is-visible',
+                menu_align: 'align-',
+                text_align: 'control-text-align'
+            },
+
+            init: function(parent) {
+                var that = _dropdown;
```

Comment:
`that` here references the main object and is used in the following conditional.